ref/src/main/java/io/smallrye/common/ref/References.java

```diff
@@ -31,7 +31,16 @@ static ReferenceQueue<Object> getReaperQueue() {
             if (isBuildTime()) {
                 // do nothing (class should be reinitialized)
             } else {
-                final PrivilegedAction<Void> action = () -> startThreadAction(1);
+                final PrivilegedAction<Void> action = new PrivilegedAction<>() {
+                    @Override
+                    public Void run() {
+                        final ReaperThread thr = new ReaperThread();
+                        thr.setName("Reference Reaper #1");
+                        thr.setDaemon(true);
+                        thr.start();
+                        return null;
+                    }
+                };
```

Comment:
```suggestion
                final PrivilegedAction<Void> action = () -> {
                    final ReaperThread thr = new ReaperThread();
                    thr.setName("Reference Reaper #1");
                    thr.setDaemon(true);
                    thr.start();
                    return null;
                };
```

Reply:
I've avoided this because of saving the runtime bytecode generation of lambdas